docs/advanced/settings.md

```diff
@@ -449,6 +449,14 @@ This setting allows you to define the number of minutes Harvester waits before d
 
 Default: `30`
 
+## `support-bundle-node-timeout`
+
+_Available as of v1.4.0_
+
+This setting allows you to define the number of minutes Harvester waits before node collection is timeout. If node is timeout, Harvester still could download support bundle without node's logs. The minimum value is `30`.
```

Comment:
```suggestion
This setting allows you to define the number of minutes Harvester allows for collection of node logs. If the collection process is not completed within the allotted time, Harvester still allows you to download the support bundle (without logs). The minimum value is `30`.
```

Reply:
@jillian-maroket I think use `node logs and configs` would be better. Is it okay if I revise like below?

```suggestion
This setting allows you to define the number of minutes Harvester allows for collection of node logs and configs. If the collection process is not completed within the allotted time, Harvester still allows you to download the support bundle (without node logs and configs). The minimum value is `30`.
```

Reply:
@Yu-Jack "configs" - Do you mean all settings in the Harvester configuration file (listed [here](https://docs.harvesterhci.io/v1.4/install/harvester-configuration) and [here](https://docs.harvesterhci.io/v1.4/advanced/index/)? We should as specific as possible to help the user set an appropriate value.

Reply:
It includes the Harvester and RKE2 Configuration in general. This is an example.

![image](https://github.com/harvester/docs/assets/6960289/75408177-5efa-4841-8924-a34bbec26115)

Reply:
Add some words in first sentence.

```suggestion
This setting allows you to define the number of minutes Harvester allows for collection of node logs and configs in support bundle. If the collection process is not completed within the allotted time, Harvester still allows you to download the support bundle (without node logs and configs). The minimum value is `30`.
```

Reply:
Thank you for clarifying. Please check if the revised description is appropriate. 

This setting allows you to define the number of minutes Harvester allows for collection of logs (nodes) and configurations (Harvester and guest clusters). If the collection process is not completed within the allotted time, Harvester still allows you to download the support bundle (without the uncollected data). The minimum value is `30`.

Reply:
Thanks for revising that. Is it appropriate?

This setting allows you to define the number of minutes Harvester allows for collection of logs and configurations (Harvester and Cluster) on the nodes for the support bundle. If the collection process is not completed within the allotted time, Harvester still allows you to download the support bundle (without the uncollected data). The minimum value is `30`.

Reply:
Okay, I think it's just a Harvester without guest cluster here, thanks! Let me revise it.